src/lib/Components/ArtworkTileRail/ArtworkTileRail.tsx

```diff
@@ -25,35 +20,21 @@ export const ArtworkTileRailContainer: React.FC<{
     <View ref={navRef}>
       <FlatList
         horizontal
-        style={{ borderRadius: 2, overflow: "hidden" }}
         ItemSeparatorComponent={() => <Spacer width={15}></Spacer>}
         showsHorizontalScrollIndicator={false}
         data={artworks}
         initialNumToRender={5}
         windowSize={3}
         renderItem={({ item }) => (
-          <ArtworkCard
+          <ArtworkTileRailCard
             onPress={() => {
               tracking.trackEvent(tappedArtworkGroupThumbnail(contextModule, item!.node!.internalID, item!.node!.slug))
               SwitchBoard.presentNavigationViewController(navRef.current!, item?.node?.href! /* STRICTNESS_MIGRATION */)
             }}
-          >
-            <Flex>
-              <OpaqueImageView
-                imageURL={(item?.node?.image?.imageURL ?? "").replace(":version", "square")}
-                width={SMALL_TILE_IMAGE_SIZE}
-                height={SMALL_TILE_IMAGE_SIZE}
-              />
-              <Box mt={1} width={SMALL_TILE_IMAGE_SIZE}>
-                <Sans size="3t" weight="medium" numberOfLines={1}>
-                  {item?.node?.artistNames}
-                </Sans>
-                <Sans size="3t" color="black60" numberOfLines={1}>
-                  {item?.node?.saleMessage}
-                </Sans>
-              </Box>
-            </Flex>
-          </ArtworkCard>
+            imageURL={item?.node?.image?.imageURL ?? ""}
```

Comment:
`imageURL` is nullable so I think we can remove the null coalesce:
```suggestion
            imageURL={item?.node?.image?.imageURL}
```

Reply:
Good call, I'll make this change!